packages/neuron-ui/src/services/UILayer.ts

```diff
@@ -36,27 +36,31 @@ export const getWallets = () => {
 export const createWallet = (wallet: { name: string; mnemonic: any; password: string }) =>
   UILayer.send(Channel.CreateWallet, wallet)
 
-export const deleteWallet = (walletID: string, password: string, valid: any) => {
+export const deleteWallet = (walletID: string, password: string, callback: any) => {
   UILayer.on(Channel.DeleteWallet, (_e: any, args: Response<string>) => {
-    console.error(args)
     if (args.result) {
       getWallets()
     }
-    valid(args)
+    callback(args)
```

Comment:
I'd like to know what the callback is going to do. It seems to verify the args in the previous commit, but the name `valid(args)` is unclear. If it does the verification work, it could be named as `verify(args)`, `isValid(args)`.

Reply:
ok

Reply:
It handles the result from core.named as `handleResult`?